lib/routes/account.js

```diff
@@ -974,6 +974,31 @@ module.exports = (log, db, mailer, Password, config, customs, checkPassword, pus
           }
         }
 
+        function sendVerifyLoginPush(devices) {
+          if (doSigninConfirmation) {
+            log.info({
+              op: 'account.signin.confirm.push.start',
+              uid: accountRecord.uid,
+              tokenVerificationId: tokenVerificationId
+            })
+
+            return push.notifyVerifyLogin(accountRecord.uid, devices)
+          }
+        }
+
+        function sendVerifyLogin() {
+          return db.devices(accountRecord.uid).then(
+            function (devices) {
+              //filter the devices variable to devices which can receive a push notification
```

Comment:
The `db.devices` promise will return an array of devices, here's what a device object looks like: https://github.com/mozilla/fxa-auth-server/blob/master/lib/db.js#L451-L466

As for the best way to filter the devices, perhaps others (@philbooth) might have better ideas, but I'd consider checking `device.type` for `"desktop"`, and `device.uaBrowserVersion`... though the version is probably a string, like `"54.0.1"`, so it'd need to be able to compare major versions...